build.assets/charts/Dockerfile

```diff
@@ -1,12 +1,24 @@
-FROM ubuntu:18.10
+FROM ubuntu:18.04
 
-RUN apt-get update && apt-get install -y \
-    dumb-init \
- && rm -rf /var/lib/apt/lists/*
+# Make sure apt is updated and image is built with the latest minor release.
+RUN apt-get update && apt-get upgrade -y
 
-# Bundle teleport and control binary
+# Install dumb-init, this is to make sure that signals are handled correctly
+# and orphaned processes are cleaned up. For more details see:
+# https://github.com/Yelp/dumb-init.
+RUN apt-get install -y dumb-init
+
+# Ubuntu images don't come with any certificate authorities, they need to be
+# installed manually.
+RUN apt-get install -y ca-certificates && update-ca-certificates
+
+# Clean up /var/lib/apt/lists/* which is populated by "apt update" to reduce
+# size of image.
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
This won't work as intended.

Because docker operates on layers, the /var/lib/apt/lists files will get included in apt-get update, that layer will be saved and part of the overall image, and then the rm command will just remove the file pointers. This is why you mostly see chained apt-get commands in docker files, apt-get updates && apt-get install ... && rm -rf ... ensures the download, install, cleanup all run in the same layer, and as such the files removed don't actually add to the image size.